src/lib/components/simple/common/VerticalDraggableList.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+  import { flip } from "svelte/animate";
```

Comment:
anche qua l'indent si è sputtanato.
Magari aggiungi il file di configurazione di VSCode (tipo come è fatto in tatum) che così non ci sono più problemi con l'editor